XCUITests/NavigationTest.swift

```diff
@@ -0,0 +1,243 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+let url_1 = "www.mozilla.org"
+let url_1Label = "Internet for people, not profit — Mozilla"
+let url_1Value = "mozilla.org"
+
```

Comment:
just for cosmetic reason, maybe we can use swift dictionary? Like:
var URL:[String:String] = ["name":"www.mozilla.org", "label":"Internet...", "value":"mozilla.org"]
then call it by URL[name]
That or using struct like 
struct URL {
   var name: String
   var label: String
   var value: String
}